lib/xcskarel/server.rb

```diff
@@ -125,6 +136,10 @@ def post_endpoint(endpoint, body)
       call_endpoint("post", endpoint, body)
     end
 
+    def delete_endpoint(endpoint, body)
```

Comment:
I don't think we'll ever need the `body` here, feel free to remove it and pass in nil.